easybuild/easyconfigs/b/Beast/Beast-1.10.1-intel-2018a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
```

Comment:
@boegel please re-instate the original credit

Reply:
@verdurin fixed with 43a75d405